src/ui/public/agg_types/buckets/terms.js

```diff
@@ -130,6 +132,13 @@ define(function (require) {
 
             let orderAgg = agg.params.orderAgg || vis.aggs.getResponseAggById(agg.params.orderBy);
 
+            const orderBy = orderAgg.type.name;
+            const sortOrder = agg.params.order.val;
+            if (orderBy === 'count' && sortOrder === 'asc') {
+              const notify = new Notifier();
```

Comment:
Can we keep this at a higher scope? No reason to recreate it all the time. Perhaps you could also use this as a way to debounce the calls...